reference/carts.v3.yml

```diff
@@ -1817,10 +1817,28 @@ components:
               list_price:
                 type: string
                 description: 'Price of the item. With or without tax, depending on your store’s setup. Optional price override.'
```

Comment:
Recommend changing the description to say the following: "Specifies the price of the item. This value can include or exclude tax, depending on the setup of your store."